subworkflows/nf-core/vcf_annotate_snpeff/main.nf

```diff
@@ -0,0 +1,28 @@
+//
+// Run SNPEFF to annotate VCF files
+//
+
+include { SNPEFF           } from '../../../modules/nf-core/snpeff/main.nf'
+include { TABIX_BGZIPTABIX } from '../../../modules/nf-core/tabix/bgziptabix/main.nf'
+
+workflow VCF_ANNOTATE_SNPEFF {
+    take:
+    vcf          // channel: [ val(meta), vcf ]
+    snpeff_db    //   value: db version to use
+    snpeff_cache //    path: /path/to/snpeff/cache (optionnal)
+
+    main:
+    ch_versions = Channel.empty()
+
+    SNPEFF(vcf, snpeff_db, snpeff_cache)
+    TABIX_BGZIPTABIX(SNPEFF.out.vcf)
```

Comment:
Personally, I'd vote for editing `snpeff` to make it output a compressed vcf + index, but that's just nitpicking

Reply:
My long term plan would be to actually scatter/gather the annotation, and gzip index the end VCF.